### ISSUE TITLE
Create a way to import test data into Prometheus for testing alerts and queries

### ISSUE THREAD

**Issue by mattfredericksen** [mattfredericksen/CSCE-3444-Group#12]
https://prometheus.io/docs/instrumenting/pushing/
https://github.com/prometheus/client_python#exporting-to-a-pushgateway
https://github.com/prometheus/pushgateway/blob/master/README.md

**Comment by EronWright** [comment]
@mattfredericksen did you consider restoring prometheus from a snapshot? ([ref](https://prometheus.io/docs/prometheus/latest/storage/))

**Comment by mattfredericksen** [comment]
Hello @EronWright, I never expected to see someone comment here. (edit: ah, I see that when I made this repo public, my comments were linked in prometheus's repo).

Unfortunately, I don't think using snapshots would have worked for my needs. This was a one-semester university project. The primary feature I wanted to test was forecasting trends and generating alerts when those trends met certain conditions. From what I recall, prometheus is typically used for short-term systems monitoring. In my case, I wanted to store ~5 years of data, and make predictions based on the last month of recorded data. 

With the relatively short time I had available for this project, I could not record sufficient data to reliably test all of the features I wanted to implement, and so I also could not obtain a snapshot from which to restore prometheus. Hence the motivation to inject test data.

Let me know if there's something I misunderstood about your suggestion. I enjoyed working with prometheus.
